tests/test_self.py

```diff
@@ -122,11 +121,11 @@ def _display(self, layout: "Section") -> None:
         pass
 
     @property
-    def out(self) -> StringIO:
+    def out(self) -> TextIO:  # type: ignore # Property is not the same as BaseLinter's attr
```

Comment:
@cdce8p You also commented on this in #4954, see https://github.com/PyCQA/pylint/pull/4954#discussion_r702472666 and https://github.com/PyCQA/pylint/pull/4954#discussion_r702472684
However, this still is an issue. `mypy` maintainers seem to suggest not doing this kind of stuff (see https://github.com/python/mypy/issues/8185). Do we want to refactor (can we?) or is adding the `type: ignore` enough. For some reason for the one below I needed two comments.

Reply:
I've come across https://github.com/python/mypy/issues/8185 in another project already. Was doing something similar 😅 `type: ignore` would be fine IMO. See also https://github.com/python/mypy/issues/7994 which suggest to use `type: ignore` to silence the error.

```suggestion
    def out(self) -> TextIO:  # type: ignore[override]
```